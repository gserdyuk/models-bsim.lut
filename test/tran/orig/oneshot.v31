** One-Shot Trigger (Tran): Benchmarking Implementation of BSIM3 by Jane Xi 5/16/2005.

Md1 4 in Vdd Vdd  P1 W=3.6u L=1.2u
Md2 4 in 0 0 N1 W=1.8u L=1.2u
c4 4 0 30f
Md3 A 4 Vdd Vdd  P1 W=3.6u L=1.2u
Md4 A 4 0 0 N1 W=1.8u L=1.2u
ca a 0 30f

M1 Anot A Vdd Vdd  P1 W=3.6u L=1.2u
M2 Anot A 0 0 N1 W=1.8u L=1.2u

M3 Bnot in Vdd Vdd  P1 W=3.6u L=1.2u
M4 Bnot in 0 0 N1 W=1.8u L=1.2u

M5 AorBnot 0 Vdd Vdd P1 W=1.8u L=3.6u
M6 AorBnot in 1 0 N1 W=1.8u L=1.2u
M7 1 Anot 0 0 N1 W=1.8u L=1.2u

M8 Lnot 0 Vdd Vdd P1 W=1.8u L=3.6u
M9 Lnot Bnot 2 0 N1 W=1.8u L=1.2u
M10 2 A 0 0 N1 W=1.8u L=1.2u

M11 out 0 Vdd Vdd P1 W=3.6u L=3.6u
M12 out AorBnot 3 0 N1 W=1.8u L=1.2u
M13 3 Lnot 0 0 N1 W=1.8u L=1.2u

Vcc vdd 0 1.8
vin in 0 pulse 0 1.8 1ns .1ns .1ns .8ns 5ns

.include modelcard.nmos
.include modelcard.pmos

.tran 1ns 10ns
.print tran in out
.option post

.end
