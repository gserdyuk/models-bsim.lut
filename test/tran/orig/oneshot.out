Note: can't find init file.
Warning: vin: no DC value, transient time 0 value used

Circuit: ** One-Shot Trigger (Tran): Benchmarking Implementation of BSIM3 by Jane Xi 5/16/2005.

Warning: Pd = 0 is less than W.
Warning: Ps = 0 is less than W.
Warning: Pd = 0 is less than W.
Warning: Ps = 0 is less than W.
Warning: Pd = 0 is less than W.
Warning: Ps = 0 is less than W.
Warning: Pd = 0 is less than W.
Warning: Ps = 0 is less than W.
Circuit: ** One-Shot Trigger (Tran): Benchmarking Implementation of BSIM3 by Jane Xi 5/16/2005.
Date: Thu Jul 28 21:10:07  2005


** One-Shot Trigger (Tran): Benchmarking Implementation of BSIM3 by Jane Xi 5/16/2005.
Transient Analysis  Thu Jul 28 21:10:07  2005
--------------------------------------------------------------------------------
Index   time            in              out             
--------------------------------------------------------------------------------
0	0.000000e+00	0.000000e+00	8.397409e-01	
1	2.000000e-12	0.000000e+00	8.397332e-01	
2	4.000000e-12	0.000000e+00	8.397327e-01	
3	8.000000e-12	0.000000e+00	8.397320e-01	
4	1.600000e-11	0.000000e+00	8.397320e-01	
5	3.200000e-11	0.000000e+00	8.397332e-01	
6	6.400000e-11	0.000000e+00	8.397342e-01	
7	1.280000e-10	0.000000e+00	8.397349e-01	
8	2.560000e-10	0.000000e+00	8.397352e-01	
9	4.560000e-10	0.000000e+00	8.397352e-01	
10	6.560000e-10	0.000000e+00	8.397351e-01	
11	8.560000e-10	0.000000e+00	8.397351e-01	
12	1.000000e-09	0.000000e+00	8.397346e-01	
13	1.010000e-09	1.800000e-01	8.419163e-01	
14	1.030000e-09	5.400000e-01	8.466081e-01	
15	1.070000e-09	1.260000e+00	8.632560e-01	
16	1.100000e-09	1.800000e+00	8.678726e-01	
17	1.104262e-09	1.800000e+00	8.647707e-01	
18	1.112785e-09	1.800000e+00	8.579861e-01	
19	1.129831e-09	1.800000e+00	8.433689e-01	
20	1.163925e-09	1.800000e+00	8.107047e-01	
21	1.214951e-09	1.800000e+00	7.711529e-01	
22	1.317002e-09	1.800000e+00	7.666830e-01	
23	1.415326e-09	1.800000e+00	8.686463e-01	
24	1.611972e-09	1.800000e+00	1.099349e+00	
25	1.811972e-09	1.800000e+00	1.256252e+00	
26	1.900000e-09	1.800000e+00	1.316613e+00	
27	1.910000e-09	1.620000e+00	1.321281e+00	
28	1.930000e-09	1.260000e+00	1.331131e+00	
29	1.970000e-09	5.400000e-01	1.354555e+00	
30	2.000000e-09	0.000000e+00	1.376749e+00	
31	2.004069e-09	0.000000e+00	1.380440e+00	
32	2.012208e-09	0.000000e+00	1.387884e+00	
33	2.028486e-09	0.000000e+00	1.402687e+00	
34	2.061040e-09	0.000000e+00	1.431353e+00	
35	2.118077e-09	0.000000e+00	1.474881e+00	
36	2.232151e-09	0.000000e+00	1.539246e+00	
37	2.349596e-09	0.000000e+00	1.576483e+00	
38	2.498448e-09	0.000000e+00	1.592371e+00	
39	2.698448e-09	0.000000e+00	1.581858e+00	
40	2.898448e-09	0.000000e+00	1.550551e+00	
41	3.098448e-09	0.000000e+00	1.508850e+00	
42	3.298448e-09	0.000000e+00	1.461497e+00	
43	3.498448e-09	0.000000e+00	1.412163e+00	
44	3.698448e-09	0.000000e+00	1.362753e+00	
45	3.898448e-09	0.000000e+00	1.314438e+00	
46	4.098448e-09	0.000000e+00	1.267380e+00	
47	4.298448e-09	0.000000e+00	1.221949e+00	
48	4.498448e-09	0.000000e+00	1.178141e+00	
49	4.698448e-09	0.000000e+00	1.135895e+00	
50	4.898448e-09	0.000000e+00	1.095234e+00	
51	5.098448e-09	0.000000e+00	1.056420e+00	
52	5.298448e-09	0.000000e+00	1.020045e+00	
53	5.498448e-09	0.000000e+00	9.869489e-01	
54	5.698448e-09	0.000000e+00	9.579269e-01	

Index   time            in              out             
--------------------------------------------------------------------------------
55	5.898448e-09	0.000000e+00	9.335683e-01	
56	6.000000e-09	0.000000e+00	9.230822e-01	
57	6.010000e-09	1.800000e-01	9.236453e-01	
58	6.030000e-09	5.400000e-01	9.249859e-01	
59	6.070000e-09	1.260000e+00	9.328911e-01	
60	6.100000e-09	1.800000e+00	9.321754e-01	
61	6.104272e-09	1.800000e+00	9.291879e-01	
62	6.112817e-09	1.800000e+00	9.226992e-01	
63	6.129907e-09	1.800000e+00	9.089150e-01	
64	6.164086e-09	1.800000e+00	8.777825e-01	
65	6.215195e-09	1.800000e+00	8.390332e-01	
66	6.317415e-09	1.800000e+00	8.357927e-01	
67	6.415969e-09	1.800000e+00	9.253601e-01	
68	6.613077e-09	1.800000e+00	1.121499e+00	
69	6.813077e-09	1.800000e+00	1.270133e+00	
70	6.900000e-09	1.800000e+00	1.328253e+00	
71	6.910000e-09	1.620000e+00	1.332746e+00	
72	6.930000e-09	1.260000e+00	1.342335e+00	
73	6.970000e-09	5.400000e-01	1.365108e+00	
74	7.000000e-09	2.442491e-15	1.386759e+00	
75	7.004069e-09	0.000000e+00	1.390349e+00	
76	7.012208e-09	0.000000e+00	1.397618e+00	
77	7.028485e-09	0.000000e+00	1.412068e+00	
78	7.061040e-09	0.000000e+00	1.440166e+00	
79	7.118077e-09	0.000000e+00	1.483113e+00	
80	7.232152e-09	0.000000e+00	1.547248e+00	
81	7.349598e-09	0.000000e+00	1.584575e+00	
82	7.498453e-09	0.000000e+00	1.600345e+00	
83	7.698453e-09	0.000000e+00	1.589312e+00	
84	7.898453e-09	0.000000e+00	1.557374e+00	
85	8.098453e-09	0.000000e+00	1.515062e+00	
86	8.298453e-09	0.000000e+00	1.467150e+00	
87	8.498453e-09	0.000000e+00	1.417326e+00	
88	8.698453e-09	0.000000e+00	1.367489e+00	
89	8.898453e-09	0.000000e+00	1.318691e+00	
90	9.098453e-09	0.000000e+00	1.271343e+00	
91	9.298453e-09	0.000000e+00	1.225661e+00	
92	9.498453e-09	0.000000e+00	1.181640e+00	
93	9.698453e-09	0.000000e+00	1.139206e+00	
94	9.898453e-09	0.000000e+00	1.098363e+00	
95	1.000000e-08	0.000000e+00	1.078331e+00	

CPU time since last call: 0.440 seconds.

Total CPU time: 0.440 seconds.

Current dynamic memory usage = 327680,
Dynamic memory limit = -18809400.

